### ISSUE TITLE
Problem with start

### ISSUE THREAD

**Issue by wcizmowski** [cytopia/devilbox#454]
#### ISSUE TYPE

 - Question

#### SUMMARY
Fatal error: Uncaught RedisException: Connection refused in /var/www/default/include/lib/container/Redis.php:42 Stack trace: #0 /var/www/default/include/lib/container/Redis.php(42): Redis->connect('redis', 6379, 0.5, '') #1 /var/www/default/include/lib/container/BaseClass.php(35): devilbox\Redis->__construct('redis', Array) #2 /var/www/default/config.php(146): devilbox\BaseClass::getInstance('redis') #3 /var/www/default/htdocs/index.php(20): loadClass('Redis') #4 {main} thrown in /var/www/default/include/lib/container/Redis.php on line 42

#### Goal

In my environment, I get last changes from github, try start docker containers.....

**Comment by dvlbot** [comment]
**Thank you for submitting an issue!**

It looks like you did not choose an appropriate **[Issue Type](https://github.com/cytopia/devilbox/issues/new/choose)** while creating this issue and therefore it cannot be correctly assigned.

1. Follow this [link](https://github.com/cytopia/devilbox/issues/new/choose)
2. Select issue type
3. Copy the issue text
4. Replace your original post with the copied text
5. Fill in all required details
6. In case of bug reports, ensure it is reproducible by provided details

Beep beep, I am a bot - do not reply to me.

**Comment by wcizmowski** [comment]
I don't think that is bug, if I start all container /with redis/, then everything is ok. But... maybe bug, because some user want run without redis...

**Comment by cytopia** [comment]
I have no idea where you hit this bug, nor what versions or settings you have customized or what container are running. Can you please be a bit more specific on this one.

**Comment by cytopia** [comment]
@wcizmowski could you please clarify the issue.

**Comment by cytopia** [comment]
I will close this issue for the following reasons:

* wrong issue type
* no information to reproduce
* no information what this is actually about
* no response

Feel free to provide required information and I will re-open

**Comment by KubaChoice** [comment]
Hi, FYI
I did stuck with the same problem. 

1) docker-compose up httpd php mysql - this enforced Exception above
2) docker-compose up httpd php mysql redis - no exception / maybe it works somehow, but not for me

In my case, I wanted to run Devilbox tools, on different port that 80.

What is weird and must be somehow related is that my problem was solved when I added  own (company) DNS server to BIND_DNS_RESOLVER=8.8.8.8,8.8.4.4,xxx.xxx.xxx.xxx

After that I was able to run Devilbox tools on custom port with even no redis service in docker-compose command (docker-compose up httpd php mysql works fine now)

Maybe, information will help somebody.
